cmd/valiant/cmd.go

```diff
@@ -12,7 +12,7 @@ func main() {
 	app.Name = "valiant"
 	app.Usage = "Validate request and responses to a web site."
 
-	app.Commands = []cli.Command{
+	app.Commands = []*cli.Command{
```

Comment:
Shouldnt we vendor cli ? If cli package is updated in future again then these changes will break again.

Reply:
We can do that too. But we need to do it in external admin

Reply:
That can be added as tech debt.

Reply:
External admin does not have a dependency on cli but valiant has. Its version should be fixed in valiant. We might want to use a different version of cli in external admin according to our needs. For now I guess it should be fine.